navigation/screens/facility-profile.tsx

```diff
@@ -19,125 +16,53 @@ import { StarRatingDisplay } from "react-native-star-rating-widget";
+
 const windowHeight = Dimensions.get("window").height;
 
 function CollapseView() {
-  const [collapsed, setCollapsed] = useState(true);
-  const [maxLines, setMaxLines] = useState(2);
-  const animationHeight = useRef(new Animated.Value(0)).current;
-
-  /**get all reviews*/
-  // const handleReviews = async () => {
-  //   try {
-  //     const reviews = await getAllReviews(bathroomID);
-  //   } catch (error) {
-  //     console.error("Error retrieving reviews:", error);
-  //   }
-  // };
-
-  const toggleCollapsed = () => {
-    setCollapsed(!collapsed);
-  };
-
-  const collapseView = () => {
-    Animated.timing(animationHeight, {
-      duration: 100,
-      toValue: 80,
-      useNativeDriver: false,
-    }).start();
-  };
-
-  const expandView = () => {
-    setMaxLines(maxLineNumber);
-    Animated.timing(animationHeight, {
-      duration: 1000,
-      toValue: 1000,
-      useNativeDriver: false,
-    }).start();
-  };
-
-  useEffect(() => {
-    if (collapsed) {
-      collapseView();
-    } else {
-      expandView();
-    }
-  }, [collapsed]);
-
-  const changeText = () => {
-    if (collapsed) {
-      return "Expand";
-    } else {
-      return "Collapse";
-    }
-  };
-
-  // navigation
-  const navigation = useNavigation<ScreenNavigationProp>();
+  type onPressCallback = () => void;
+  function renderViewMore(onPress: onPressCallback) {
+    return renderPressableText("View more", onPress);
+  }
+  function renderViewLess(onPress: onPressCallback) {
+    return renderPressableText("View less", onPress);
+  }
+  function renderPressableText(content: string, onPress: onPressCallback) {
+    return (
+      <Text onPress={onPress} style={{ color: "#6da798" }}>
+        {content}
+      </Text>
+    );
+  }
 
   return (
-    <View style={{ overflow: "hidden", backgroundColor: "none" }}>
-      <Animated.View style={{ maxHeight: animationHeight }}>
-        <Text
-          style={[
-            styles.paragraph,
-            { backgroundColor: "#CDEEEA", padding: 20 },
-          ]}
-          numberOfLines={maxLines}
-        >
-          Hours: ##:## - ##:## {"\n"}
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
-          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-          aliquip ex ea commodo consequat. Duis aute irure dolor in
-          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-          culpa qui officia deserunt mollit anim id est laborum.
-        </Text>
-      </Animated.View>
-      <LinearGradient
-        // Button Linear Gradient
-        colors={["#6da798", "#40a4a9"]}
-        end={{ x: 0.1, y: 0.2 }}
-        style={styles.button}
+    <View
+      style={{
+        overflow: "hidden",
+        backgroundColor: "none",
+        marginHorizontal: "3%",
```

Comment:
Margin for this looks a little small. Can this be increased to have more breathing room?

Reply:
increased to 6